metagraph/src/graph/representation/succinct/dbg_succinct.hpp

```diff
@@ -191,6 +200,53 @@ class DBGSuccinct : public DeBruijnGraph {
     std::unique_ptr<mtg::kmer::KmerBloomFilter<>> bloom_filter_;
 };
 
+inline DBGSuccinct::node_index
+DBGSuccinct::row_diff_successor(node_index node, const bit_vector &rd_succ) const {
```

Comment:
why not make this a simple wrapper for `call_row_diff_successors`?

Reply:
this thing calls just one successor and `call_row_diff_successors` calls all of them.